packages/docusaurus/templates/classic/pages/s.module.css

```diff
@@ -0,0 +1,176 @@
+:root {
+  --first-screen-main-section-height: calc((100vh - 60px - 60px) * .6);
+  --first-screen-secondary-section-height: calc((100vh - 60px - 60px) * .4);
+}
+
+.header {
+  color: white;
+  padding: 1rem 0;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+}
+
+.buttons  {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.buttons iframe {
+  margin-left: 1rem;
+}
+.logo {
+  position: absolute;
+  bottom: -80px;
+  left: 20px;
+  animation-duration: 2s;
+  animation-name: jackInTheBox;
+}
+@media screen and (min-height: 900px) {
+  .buttons {
+    margin-top: 2rem;
+  }
+}
+@media screen and (max-width: 400px) {
+  .header {
+    /* display mainly the header section */
+    height: calc(100vh - 120px);
+  }
+}
+@media screen and (max-width: 799px) {
+.header {
+    /* display just one feature */
+    height: calc(100vh - 310px);
+  }
+}
+@media screen and (min-width: 800px) { 
+  .header {
+    padding: 4rem 0;
+    height: var(--first-screen-main-section-height);
+  }
+}
+
+@keyframes jackInTheBox {
```

Comment:
This is very specific to Docusaurus' mascot logo and might not work well with all logos. Let's remove this animation in the template.